.eslintrc.js

```diff
@@ -70,6 +71,10 @@ module.exports = {
   overrides: [
     {
       files: ['lib/**/*.js'],
+      excludedFiles: [
+        'lib/core/reporters/**/*.js',
+        'lib/**/*-after.js'
+      ],
```

Comment:
What is this for? IIRC ESLint settings cascade, so the overrides below will take effect.

Reply:
That's what I figured too. I tried a couple of different ways. Couldn't get it to work.

Reply:
I do not believe you.

```
➜  eslint-test cat lib/a.js
console.log('Hello')
➜  eslint-test cat lib/b.js
console.log('World')
➜  eslint-test cat .eslintrc.json
{
  "rules": {
    "no-console": "error"
  },
  "overrides": [
    {
      "files": "lib/b.js",
      "rules": {
        "no-console": "off"
      }
    }
  ]
}
➜  eslint-test npx eslint lib/*.js

/Users/stephen/dev/src/github.com/dequelabs/eslint-test/lib/a.js
  1:1  error  Unexpected console statement  no-console

✖ 1 problem (1 error, 0 warnings)

➜  eslint-test
```